qiskit/algorithms/phase_estimators/iphase_estimation.py

```diff
@@ -0,0 +1,193 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""The Iterative Quantum Phase Estimation Algorithm."""
+
+
+from typing import Optional, Union
+import numpy
+import qiskit
+from qiskit.circuit import QuantumCircuit, QuantumRegister
+from qiskit.circuit.classicalregister import ClassicalRegister
+from qiskit.providers import BaseBackend, Backend
+from qiskit.utils import QuantumInstance
+from qiskit.algorithms.algorithm_result import AlgorithmResult
+
+
+class IPhaseEstimation:
```

Comment:
Can we call these algorithms `IterativePhaseEstimation` to be in sync with `IterativeAmplitudeEstimation`?

Reply:
Yes